### PR TITLE
Person Draw improvements

### PR DESCRIPTION
**Pull request by jplucinski**
## Description
 - Icon styling
 - Added '+' sign for items input
 - Item delete does not break functionality 

## Motivation and Context
Person Draw widget needs UI/UX improvements.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Cogboard Contributor License Agreement.

### CODE REVIEW

File: cogboard-webapp/src/components/widgets/dialogFields/styled.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import styled from '@emotion/styled/macro';
 import NumberInput from './NumberInput';
+import { Box } from '@material-ui/core';
 
 export const StyledNumberInput = styled(NumberInput)`
   flex-basis: calc(50% - 18px);
 `;
+
+export const StyledWrapper = styled(Box)`
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/components/widgets/dialogFields/styled.js]: For me better name would be `FlexBox`

File: cogboard-webapp/src/components/widgets/types/PersonDrawWidget/styled.js [hunk 2]
```diff
@@ -2,6 +2,6 @@ import styled from '@emotion/styled/macro';
 
 import { Typography } from '@material-ui/core';
 
-export const StyledTypography = styled(Typography)`
-  height: 100%;
+export const CenterHorizontal = styled(Typography)`
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/components/widgets/types/PersonDrawWidget/styled.js]: For me better name would be `CenteredTypography`

File: cogboard-webapp/src/components/widgets/dialogFields/styled.js (hunk 1 above)
Comment (jplucinski) [reply cogboard-webapp/src/components/widgets/dialogFields/styled.js]: Fixed

File: cogboard-webapp/src/components/widgets/types/PersonDrawWidget/styled.js (hunk 2 above)
Comment (jplucinski) [reply cogboard-webapp/src/components/widgets/types/PersonDrawWidget/styled.js]: Fixed

**Pull request merged**